### PR TITLE
MPB -> MOI

### PR DESCRIPTION
**Pull request by blegat**

### CODE REVIEW

**Comment by joaquimg** [comment]
It looks much cleaner!

File: src/ParameterJuMP.jl [hunk 1]
```diff
@@ -13,6 +16,12 @@ struct Parameter <: JuMP.AbstractJuMPScalar
     m::JuMP.Model
 end
 
+# Reference to a constraint in which the parameter has coefficient coef
+struct ParametrizedConstraintRef
+    cref::JuMP.ConstraintRef
+    coef::Float64
+end
```
Comment (joaquimg) [inline src/ParameterJuMP.jl:23]: here we only allow one parameter per constraint?

Comment (joaquimg) [reply src/ParameterJuMP.jl:23]: ok, got it! each parameter can have one coef in each constraint. Great

**Comment by blegat** [comment]
Do we merge it into master or do you prefer master to work with MPB and JuMP 0.18

**Comment by joaquimg** [comment]
I think we should target jump 0.19 indeed. Lets merge to master

**Pull request merged**